virttest/env_process.py

```diff
@@ -1679,8 +1682,10 @@ def postprocess(test, params, env):
 
     libvirtd_inst = None
     vm_type = params.get("vm_type")
-
-    cpu_family = cpu_utils.get_family() if hasattr(cpu_utils, 'get_family') else cpu_utils.get_cpu_arch()
+    try:
+        cpu_family = cpu_utils.get_family() if hasattr(cpu_utils, 'get_family') else cpu_utils.get_cpu_arch()
+    except NotImplementedError:
+        cpu_family = "unknown"
```

Comment:
Just wondering: is it not possible to simply return "unknown" cpu family and thus avoid the entire try catch phrase? It is certainly simpler than the current approach. Instead of raising the NotImplementedError, the function in question could simply return "unknown" which is more consistent with the function returning a result of what it detects rather than switching gear to raising exceptions.